config/locales/en.yml

```
@@ -234,9 +234,14 @@ en:
     new:
       title: Create a new service
       heading: Add a service
+    edit:
+      description: Edit service
```

it's primarily used as a page title
```suggestion
      title: Edit service
```

---

It is used in the title and heading

---

so why would it be called `description` if it's not used in a description at all?